portal/models/identifier.py

```diff
@@ -48,9 +48,15 @@ def from_fhir(cls, data):
     def __str__(self):
         return 'Identifier {0.use} {0.system} {0.value}'.format(self)
 
+    def __key(self):
```

Comment:
I would suggest you follow the python `dunder` pattern used for "magic methods" - that is, double underscores on both sides of key, `def __key__(self): ...`

Reply:
Got it, will do.